src/components/broadcaster/BroadcastDashBoard.tsx

```diff
@@ -17,49 +21,44 @@ const BroadcastDashboard = () => {
     setIsFirstMessage(isFirst)
   }
 
+  const handleSendNow = () => {
+    void makeBroadcast()
+  }
```

Comment:
We should create a react-query useMutation. That way you could access `pending` state.

Reply:
We don't need to as we switched to using missive's button